practice_projects/#Homework_pt.py

```diff
@@ -0,0 +1,34 @@
+#Homework pt.1
+
+Friend= "\nSoftware is my friend!\n"
+print (Friend)
```

Comment:
Note for the future:
No whitespace between print(Friend)

Also, have variables start with a lower-case so we can follow snake-case conventions

Reply:
Fixed